tests/e2e/General/HTTPTest.php

```diff
@@ -13,6 +13,8 @@ class HTTPTest extends Scope
     use ProjectNone;
     use SideNone;
 
+    // TODO: We are missing a lof of tests for each create and update endpoint that can take customID. There should be FAILED check with 409 conflict, if ID is already used.
```

Comment:
I don't think leaving a TODO for testing is appropriate. Since we're writing the logic to catch these exceptions, this PR should contain the e2e tests to validate this change.